src/styles/_article.scss

```diff
@@ -9,13 +9,21 @@
     h4,
     p {
       margin-top: 20px;
+      a {
+        color: black;
```

Comment:
We have never used black on our website. It seems odd from design view that we would start now? We have used this dark pearl black though.
`$finsec-black-pearl: #0d0d13;`